client/src/containers/CreateCardFormContainer/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import CreateCardForm from 'src/components/CreateCardForm'
+
+const axios = require('axios')
+
+class CreateCardFormContainer extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      description: '',
+    }
+  }
+
+  render = () => (
+    <CreateCardForm
+      description={this.description}
```

Comment:
`this.description` returns `undefined` all the time, `this.state.description` should be used instead 😉

Reply:
👌